web/src/components/circulargauge.js

```diff
@@ -70,7 +70,7 @@ export default class extends React.PureComponent {
               <path className="background" d={this.arc.endAngle(2 * Math.PI)()} />
               <path
                 className="foreground"
-                d={this.arc.endAngle((this.state.prevPercentage / 100) * 2 * Math.PI)()}
+                d={this.arc.endAngle((percentage / 100) * 2 * Math.PI)()}
```

Comment:
why this change?

Reply:
I noticed that the D3 interpolation animations don't always get triggered properly with new tooltips when hovering over map (determining why is a separate question which motivated me to open #2286), so I thought it would be good to have a proper fallback value in such cases.

So if the animation doesn't get triggered, it should display the end result instead of the initial state - otherwise it will always be stuck at 0% in such cases when the gauge is first rendered.

Makes sense?

Reply:
ok!